packages/types/src/serde.ts

```diff
@@ -26,7 +26,7 @@ declare global {
   /**
    * @public
    */
-  export interface ReadableStream {}
+  export interface ReadableStream<R = any> {}
```

Comment:
is this an issue if you have the dom or node types library installed?

Reply:
@kuhe sorry for the late reply

This is how it's defined both in the DOM typings:

https://github.com/microsoft/TypeScript/blob/cb44488fcec4348a448434afbf2ebcbf2b423c61/src/lib/dom.generated.d.ts#L19086

and in the Node.js typings:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3c2bac796de55bd7fe6c00bab670829efa6fbc1e/types/node/stream/web.d.ts#L173

I have had no problems using it in our project that has the node typings installed, and I just tested adding DOM to libs and it works as well 👍